test/hotspot/jtreg/runtime/jni/nativeStack/libnativeStack.c

```diff
@@ -109,7 +109,17 @@ Java_TestNativeStack_triggerJNIStackTrace
 
   warning = warn;
 
-  if ((res = pthread_create(&thread, NULL, thread_start, NULL)) != 0) {
+#ifdef AIX
```

Comment:
Yes you are right, then I would change it. 

> I'm still a little confused about this issue because we have:
> 
> ```
> ./os_cpu/aix_ppc/os_aix_ppc.cpp:size_t os::_java_thread_min_stack_allowed = 64 * K;
> ```
> 
> but the current issues suggests the pthread default of 192K is too small, so 64K is definitely too small!
> 
> I would expect we have the same problem with other tests ... though some of them may be (artificially?) restricted to run on Linux. I have a test in #14139 that will likely need the same adjustment.

For this test , the default thread stack size is defined here  : 
`./src/hotspot/os_cpu/aix_ppc/globals_aix_ppc.hpp : define_pd_global(intx, ThreadStackSize,          2048); // 0 => use system default`

`test/hotspot/jtreg/runtime/jni/getCreatedJavaVMs/TestGetCreatedJavaVMs.java `failed with an exit value 134. The test is passing when stack size is explicitly provided.

Reply:
My point is that according to the definition you should be able to specify `-Xss64k` and it will work fine. But obviously that can't be the case if the default 192K is not sufficient. So `_java_thread_min_stack_allowed` seems to be set wrong for this platform.